src/cortex_cli/cortex_cli.py

```diff
@@ -32,14 +32,15 @@
                              logout_request, refresh_request,
                              time_left_seconds)
 from cortex_cli.circuit import validate_circuit
-from cortex_cli.token_manager import (check_daemon, daemonize_token_manager,
-                                      kill_by_pid)
+from cortex_cli.token_manager import (check_token_manager,
+                                      daemonize_token_manager, kill_by_pid,
+                                      start_token_manager)
 from cortex_cli.utils import read_file, read_json
 
 HOME_PATH = str(Path.home())
 DEFAULT_CONFIG_PATH = f'{HOME_PATH}/.config/iqm-cortex-cli/config.json'
 DEFAULT_TOKENS_PATH = f'{HOME_PATH}/.cache/iqm-cortex-cli/tokens.json'
-BASE_URL = 'https://auth.demo.qc.iqm.fi'
+AUTH_SERVER_URL = 'https://auth.demo.qc.iqm.fi'
```

Comment:
why do we have this specific URL as default?

Reply:
Cortex CLI being a publicly available project, I thought it makes sense to provide the "publicly available" URL.

Reply:
Well, I would prefer not to have default at all and always force the users to set it, instead of having great quantum confusion :D. This URL is public, yes, but this is not an intended use for it.

Reply:
Agreed. Fixed.